Gemfile

```diff
@@ -27,7 +27,8 @@ group :development do
 end
 
 group :system_tests do
-  gem 'beaker'
+  gem 'beaker', :path => '/home/thanos/Work/beaker'
```

Comment:
needs to be removed; and thanos did nothing wrong ;)

Reply:
@heliocentric Indeed....

Reply:
This comment was....inevitable....